tests/migrations/test_operations.py

```diff
@@ -4107,6 +4107,64 @@ def test_add_constraint(self):
             definition[2], {"model_name": "Pony", "constraint": gt_constraint}
         )
 
+    @skipUnlessDBFeature("supports_table_check_constraints")
+    def test_create_model_constraint_percent_escaping(self):
```

Comment:
This is a copy of the `test_add_constraint_percent_escaping` test below that makes two modifications

1. It uses `CreateModel` instead of `AddConstraint`
2. It uses `db_default` instead of `default` as the latter is never considered when adding a column (there's no existing rows to backfill)